### PR TITLE
Allow department_officer role to access GET /users

### PR DESCRIPTION
**Pull request by p16i**

### CODE REVIEW

**Comment by supasate** [comment]
Fix eslint. (Try `npm run test` or `yarn test` in local machine first.

File: src/services/user/hooks/index.js [hunk 1]
```diff
@@ -19,7 +20,13 @@ exports.before = {
     auth.populateUser(),
     auth.restrictToAuthenticated(),
     auth.restrictToRoles({
-      roles: [SUPER_ADMIN, ORGANIZATION_ADMIN, EXECUTIVE_ADMIN, DEPARTMENT_HEAD],
```
Comment (parnurzeal) [inline src/services/user/hooks/index.js:22]: Please also add `DEPARTMENT_OFFICER` to get method below

File: test/services/user/index.test.js [hunk 2]
```diff
@@ -167,12 +182,54 @@ describe('user service', () => {
...
+                'department_head@youpin.city',
+                'organization_admin@youpin.city',
+                'super_admin@youpin.city',
+              ];
+
+              for (let i = 0; i < userDataList.length; i++) {
+                expect(userEmails).to.include(userDataList[i].email);
+              }
+              // also check response does not contain password
+              expect(userDataList).to.not.have.keys('password');
```
Comment (parnurzeal) [inline test/services/user/index.test.js:201]: Confirming. Does this not.have.keys also work with array ("userDataList")?

Comment (p16i) [reply test/services/user/index.test.js:201]: I think so. This code is copied from the existing tests.

Comment (parnurzeal) [reply test/services/user/index.test.js:201]: Can you help recheck, plsss? Thx :)

Comment (p16i) [reply test/services/user/index.test.js:201]: Ok, It's not working and will be fixed in #294

**Comment by parnurzeal** [comment]
LGTM! Thanks!

**Pull request merged**